src/galaxy.cpp

```diff
@@ -541,7 +550,9 @@ void GalaxyView::clickColoniesButton(int x, int y, int arg) {
 }
 
 void GalaxyView::clickPlanetsButton(int x, int y, int arg) {
-	new MessageBoxWindow(this, "Not implemented yet");
+	GuiView *view = NULL;
+	view = new PlanetsListView(_game);
+	gui_stack->push(view);
```

Comment:
When you're not juggling multiple class instances in an exception handler, you can do simply `gui_stack->push(new PlanetListView(_game));`. And you should also pass `_activePlayer`.

Reply:
Why do I need `_activePlayer`?

Reply:
So that you can filter out your own planets.

Reply:
Okay